package.json

```diff
@@ -1,31 +1,32 @@
 {
-  "name": "metalsmith-clean-css",
-  "version": "5.0.1",
-  "description": "A Metalsmith plugin to minify CSS files",
-  "keywords": [
-    "metalsmith",
-    "plugin"
-  ],
+  "name": "metalsmith-clean-css-monorepo",
+  "private": true,
   "author": "Aymeric Beaumet <aymeric@beaumet.me>",
```

Comment:
Feel free to add yourself as collaborator 👍

Reply:
I am good :D